client/src/modules/exchange/exchange.modal.html

```diff
@@ -4,6 +4,7 @@
       <li class="static" translate>EXCHANGE.TITLE</li>
       <li class="title" translate>EXCHANGE.ADDING_RATE</li>
     </ol>
+    <p ng-if="ModalCtrl.missingRates" class="missing-exchange-rate">{{ModalCtrl.missingRatesWarning}}</p>
```

Comment:
There appears to be a logic bug here where `getMissingRates()` doesn't return the correct value after setting the exchange rate.  To reproduce, follow the steps in the test case, submit a new rate, and then try to set another rate and this message will persist.

![image](https://user-images.githubusercontent.com/896472/127987037-e4ee3dd7-789b-4771-9490-029ec20d722d.png)